server/taskflows/hpccloud/taskflow/nwchem/__init__.py

```diff
@@ -210,12 +214,56 @@ def monitor_nwchem_job(task, upstream_result):
     task.throws=(Retry,),
 
     job = upstream_result['job']
-    # TODO - We are currently reaching in and used a 'private' function
     _monitor_jobs(task, cluster, [job], girder_token=girder_token, monitor_interval=30)
 
     return upstream_result
 
 @cumulus.taskflow.task
+def create_json_output(task, upstream_result):
+    task.logger.info('Converting nwchem output to json format.')
+    cluster = upstream_result['cluster']
+    job = upstream_result['job']
+    job_dir = job_directory(cluster, job)
+    out_file = '%s.out' % (job['name'])
+
+    try:
+        # Copy the nwchem output to server
+        tmp_dir = tempfile.mkdtemp()
```

Comment:
@chetnieter Sorry I didn't realize TemporaryDirectory was introduced in Python 3.